docs/platform_management_plan/project_management.rst

```diff
@@ -181,4 +181,219 @@ meetings, where such topics are discussed and decided, see `Steering committees`
 
 Planning
 ========
-coming soon
+
+Planning infrastructure
+------------------------
+`GitHub issues <https://github.com/features/issues>`_ are used to plan and to track
+work. To be able to find issues faster and to filter for them more efficiently,
+we use labels.
+
+Labels
+^^^^^^
+To facilitate the organization and tracking of tickets related to the same feature
+or topic, labels are utilized for issues and pull requests. Labels are a powerful
+feature that allows you to search and filter tickets based on specific labels, and
+you can save these filters in a *GitHub Project* view. However, it is important
+to exercise caution when creating labels to avoid confusion and ensure easy tracking.
+
+It's worth noting that labels are associated with a repository, not a *GitHub Project*.
+To create new labels in the repository requires special rights and typically only
+*project leads leads* and *committers* should have this capability.
+
+For the main *SCORE* repository, there exist already some predefined labels:
+
+* *contribution_request* label is used to identify *PRs* and *GitHub Issues* that are part
```

Comment:
consider changing "contribution_request" to "feature_request" also below in section "Planning process" because of the changed scope of a contribution_request.

Reply:
Fixed